setup.py

```diff
@@ -178,7 +178,7 @@ def check_for_openmp():
         "url": 'https://khmer.readthedocs.org/',
         "packages": ['khmer', 'khmer.tests', 'oxli'],
         "package_dir": {'khmer.tests': 'tests'},
-        "install_requires": ['screed >= 0.9', 'bz2file'],
+        "install_requires": ['screed >= 0.9', 'bz2file', 'simplesam'],
```

Comment:
I'm -0 on this as a requirement; this is all for a sandbox script that most people won't use. Can we put it in extras or somewhere else?